### PR TITLE
German translations

### PR DESCRIPTION
**Pull request by K0bin**
I translated the two strings to German.

### CODE REVIEW

**Comment by avluis** [comment]
Neat -- thank you~

**Comment by Andrew-Quebe** [comment]
Merging this now, but there may also be programmatically generated text that will need translating as well. Most of that, however, can be translated on the user side when they use the library.

**Comment by Andrew-Quebe** [comment]
@K0bin refer to issue #5 again if you want to help translate the example app strings.

**Comment by K0bin** [comment]
I'm going to do it but to be honest I don't see the point in translating the example app.

**Comment by Andrew-Quebe** [comment]
@K0bin it's not mandatory, so if you don't want to, that's fine.

File: library/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="skip_button">ÜBERSPRINGEN</string>
```
Comment (heinrichreimer) [inline library/src/main/res/values-de/strings.xml]: Using all-caps text in string resources is an anti-pattern.
Should probably be handled by the `TextView`.

**Pull request merged**